cmd/kubelet/app/server.go

```diff
@@ -255,7 +255,7 @@ is checked every 20 seconds (also configurable with a flag).`,
 				config.StaticPodURLHeader[k] = []string{"<masked>"}
 			}
 			// log the kubelet's config for inspection
-			klog.V(5).InfoS("KubeletConfiguration", "configuration", config)
+			klog.V(5).InfoS("KubeletConfiguration", "configuration", fmt.Sprintf("%#v", config))
```

Comment:
@SataQiu Would it make sense to implement String() method for KubeletConfiguration instead?

Reply:
In my opinion, this usage is just a special case, and the internal configuration objects(such as `KubeletConfiguration`、`KubeproxyConfiguration`) usually don't need to implement the `String()` method. 
For the resource API objects, we've already provided `String()` method by code generation, such as:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/apps/v1/generated.pb.go#L3361

Reply:
Thanks for the explanations. Makes sense to me.

Reply:
/sig instrumentation
/cc @serathius
I am afraid that there would be some similar problems in other places. We should take care.

Reply:
cc @pohly

Reply:
For values where structured logging (i.e. dumping it as a struct in this case) is never going to be needed, then I don't see a problem with converting to a string at the call site.

But I agree that this looks like a workaround. `KubeletConfiguration` has a `String` implementation and that implementation is broken.